### PR TITLE
All form fields with a hint message, rewording some

### PR DESCRIPTION
**Pull request by ncounter**
## What does this PR change?

$subject

## GUI diff

Before:
before

After:
after

- [x] **DONE**

## Documentation
- No documentation needed: feature not yet documented

- [x] **DONE**

## Test coverage
- No tests: feature not yet tested

- [x] **DONE**

## Links

Fixes # https://github.com/SUSE/spacewalk/issues/17509
Tracks # **add downstream PR, if any**

- [x] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Comment by ncounter** [comment]
@mcalmer I know this is trivial and you are pretty busy, but it should address directly your comments during the demo&review meeting so I felt obligated to ping you :wink:

File: web/html/src/manager/proxy/container-config.tsx [hunk 1]
```diff
@@ -215,7 +216,8 @@ export function ProxyConfig() {
         />
         <Text
           name="maxSquidCacheSize"
-          label={t("Max Squid cache size (MB)")}
+          label={t("Max Squid cache size")}
+          hint={t("The maximum value of the Squid cache In Megabytes")}
```
Comment (Etheryte) [inline web/html/src/manager/proxy/container-config.tsx:220]: `In` → `in`

File: web/html/src/manager/proxy/container-config.tsx [hunk 2]
```diff
@@ -235,6 +238,7 @@ export function ProxyConfig() {
           name="sslMode"
           label={t("SSL certificate")}
           title={t("SSL certificate")}
+          hint={"Whether the SSL certificate is going to be created or reuse an existing one"}
```
Comment (Etheryte) [inline web/html/src/manager/proxy/container-config.tsx:241]: I would suggest a slight rewording, a-la `"Whether to create an SSL certificate or reuse an existing one"`

**Review by Etheryte — APPROVED** [review]

**Review by mcalmer — APPROVED** [review]

**Review by cbosdo — APPROVED** [review]
👍🏻

**Pull request merged**